### ISSUE TITLE
remarks branch has another error

### ISSUE THREAD

**Issue by einalex** [samuelyeewl/zotrm#10]
```
Traceback (most recent call last):
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 539, in <module>
    main(args.verbose, args.landscape, args.dry_run)
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 527, in main
    backsync_papers(zot, rmapi, config, verbose=verbose, dry_run=dry_run, exclude=sent_papers)
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 377, in backsync_papers
    attachment_metadata = rmapi.stat(rm_path)
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/rmapi.py", line 113, in stat
    raise Exception("Could not parse result from stat: \n{:s}")
```
seems to be an error in rmapi.py line 108. it needs to access field 0 instead of 1 of the split.

**Comment by samuelyeewl** [comment]
I think this is fixed in 0381c35, it needed utf-8 decoding instead of ascii. Do re-open this if the problem remains.

**Comment by maximiliankir** [comment]
This issue is present again, or at least for me it is. I am using the current version of zotrm. But it seems like rmapi stat has a different response format, than expected by zotrm. 

For the subprocess call `CompletedProcess(args=['/home/kirschner/rmapi', 'stat', '/Papers/Middleware/Ruiz et al. - 2020 - Non-Blocking Synchronization Between Real-Time and'], returncode=0)` I get the following response:

```
 Time 2022-10-08T15:20:52Z Choudhary et al. - 2020 - A comprehensive survey on model compression and ac
 Time 2022-10-12T14:51:43Z TODO Backlog
 Time 2022-10-08T08:53:37Z Lebenslauf
 Time 2022-10-08T08:38:37Z honegger2013
 Time 2022-10-12T16:45:49Z Quick sheets
 Time 2022-10-09T10:11:16Z Books
 Time 2022-10-09T10:26:13Z Entertainment
 Time 2022-10-08T20:50:29Z Rethinking the Computer Chip in the Age of AI - Penn Engineering Blog
 Time 2022-10-10T09:32:21Z Bishop-Pattern-Recognition-and-Machine-Learning-2006
 Time 2022-10-09T10:11:36Z Folder 2
 Time 2022-10-08T15:14:25Z Frameworks
 Time 2022-10-08T20:56:41Z Scalable and Fully Coupled Quantum-inspired Processor Solves Optimization Problems | Tokyo University of Science
 Time 2022-10-08T15:22:57Z Middleware
 Time 2022-10-08T20:36:07Z DeepMind AI invents faster algorithms to solve tough maths puzzles
 Time 2022-10-08T15:55:17Z Tool Pipeline
 Time 2022-10-12T16:44:59Z Ruiz et al. - 2020 - Non-Blocking Synchronization Between Real-Time and
 Time 2022-10-08T15:22:06Z Notes
 Time 2022-10-12T14:39:35Z TODO Täglich
 Time 2022-10-08T20:28:25Z Fussballmanager
 Time 2022-10-09T10:11:26Z Science
 Time 2022-10-08T08:39:09Z Papers
 Time 2022-10-07T21:30:23Z Real-Time AI
 Time 2022-10-08T15:19:15Z Deng et al. - 2020 - Model Compression and Hardware Acceleration for Ne
 Time 2022-10-08T20:19:29Z Hobby Projects
 Time 2022-10-08T20:29:04Z News
 Time 2022-10-11T14:11:01Z Ray - 2022 - A review on TinyML State-of-the-art and prospects
 Time 2022-10-08T15:20:41Z Cheng et al. - 2020 - A Survey of Model Compression and Acceleration for
{
  "ID": "a2173de2-4bda-4df5-bdb7-3fbf4fb0eb31",
  "Version": 0,
  "Message": "",
  "Success": false,
  "BlobURLGet": "",
  "BlobURLGetExpires": "",
  "ModifiedClient": "2022-10-12T16:44:59Z",
  "Type": "DocumentType",
  "VissibleName": "Ruiz et al. - 2020 - Non-Blocking Synchronization Between Real-Time and",
  "CurrentPage": 0,
  "Bookmarked": false,
  "Parent": "940507f9-30f6-44df-aa3e-08eddb69fd77"
}
```

To me it seems like this doesn't match the expacted format, which is used for splitting and regex matching in Line 108 and 110 of rmapi.py:
```
stat_res_str = result.stdout.split(b'\n')[1].decode('utf-8')

stat_regex = re.compile('[{|\s](\w+):(\S*)(?=[\s|}])')
match = stat_regex.findall(stat_res_str)
```
